default/methods/task/stats/fast_stats.py

```diff
@@ -0,0 +1,25 @@
+from methods.regular.regular_api import *
+from shared.database.task.job.job import Job
+from shared.database.user import User
+from shared.database.task.task import Task
+from shared.database.annotation.instance import Instance
+
+@routes.route('/api/job/<job_id>/stat', methods=["GET"])
```

Comment:
@vitalii-bulyzhyn for permissions, here's one example https://github.com/diffgram/diffgram/blob/ba092c6f11aaab9b38fc45c358807a06fe29ff90/default/methods/task/task/task_list.py#L6 
can search that wrapper for more examples.  all routes usually need permissions - the prior `is-mailer-set` was a rare exception since it effected login